packages/integration-sdk-cli/src/commands/run.ts

```diff
@@ -52,15 +51,19 @@ export function run(): Command {
 
       const clientApiOptions = getApiClientOptions(actionCommand.opts());
       const apiClient = createApiClient(clientApiOptions);
-      log.debug(
-        `Configured JupiterOne API client. (apiBaseUrl: '${clientApiOptions.apiBaseUrl}', account: ${clientApiOptions.account})`,
-      );
 
       let logger = createIntegrationLogger({
         name: 'local',
         pretty: !options.noPretty,
       });
 
+      logger.info(
```

Comment:
Any reason we're changing from debug to info?

Reply:
Previously this was using the `log.ts` instead of the actual logger. That code looks like this:
https://github.com/JupiterOne/sdk/blob/447bba4c679b24316a16cfb86a7813d5efd19f74/packages/integration-sdk-cli/src/log.ts#L14-L29

Despite it being set at debug it would always be logged since the log levels in that file are non-functioning. I decided to just keep the same behavior by setting this one to info.